README.md

```diff
@@ -458,3 +459,6 @@ The script traverses the legal-docs directory looking for localized copies of th
 Example:
 directory provided: `/Users/test/github/mozilla/legal-docs/`
 resulting file: `assets/legal/<document_name>.<locale>.pdf`
+
+_note on legal-docs_:
+We intentionally pin to a commit and return the hash at `/__version__` because the `legal-docs/package.json` is not kept up to date.
```

Comment:
Generating these PDFs doesn't affect the hash in content-server `__version__` or vice-versa.